requests_operating_unit/models/res_users.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2015-TODAY ForgeFlow
+# - Jordi Ballester Alomar
+# Copyright 2015-TODAY Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
```

Comment:
Please change all files, to be Ecosoft (please see sample from i.e., OCA/l10n-thailand)